app/assets/javascripts/components/Tooltip.js

```diff
@@ -0,0 +1,57 @@
+define(['jquery', 'DoughBaseComponent'], function($, DoughBaseComponent) {
+  'use strict';
+
+  var Tooltip,
+      defaultConfig = {};
+
+  Tooltip = function($el, config) {
+    Tooltip.baseConstructor.call(this, $el, config, defaultConfig);
+
+    this.$button = $el.find('[data-tooltip-button]');
+    this.$content = $el.find('[data-tooltip-content]');
+    this.hiddenClass = 'js-is-hidden';
+  };
+
+  /**
+  * Inherit from base module, for shared methods and interface
+  */
+  DoughBaseComponent.extend(Tooltip);
+
+  Tooltip.componentName = 'Tooltip';
+
+  /**
+  * Initialize the component
+  */
+  Tooltip.prototype.init = function(initialised) {
+    this._initialisedSuccess(initialised);
+    this._setUpComponent();
+  };
+
+  Tooltip.prototype._setUpComponent = function() {
+    var self = this;
+
+    // Touch devices
+    $(document).on('touchend', function(e) {
+      if (e.target === self.$button[0]) {
+        if (self.$content.hasClass(self.hiddenClass)) {
+          self.$content.removeClass(self.hiddenClass);
+        } else {
+          self.$content.addClass(self.hiddenClass);
+        }
+      } else {
+        self.$content.addClass(self.hiddenClass);
+      }
```

Comment:
It might be a better idea to use focus/unfocus or blur on $button instead. It would stop this code running every time the user scrolls or interacts with anything else on the page, and would also avoid unneeded DOM changes and/or DOM queries. This can be added as a future improvement.

Reply:
Ticket created to complete this work
https://moneyadviceservice.tpondemand.com/entity/9224-optimise-touch-device-js